src/libcore/convert.rs

```diff
@@ -1,26 +1,25 @@
-//! Traits for conversions between types.
 //!
-//! The traits in this module provide a general way to talk about conversions
-//! from one type to another. They follow the standard Rust conventions of
-//! `as`/`into`/`from`.
+//! The traits in this module provide a way to convert from one type to another type.
```

Comment:
So, I can appreciate that elaborating here makes sense, but it doesn't fit in with the rest of the style of the docs. These summary lines appear in places like here: 

![image](https://user-images.githubusercontent.com/27786/54935219-d9373380-4eed-11e9-83e5-ca4b39a03a13.png)

Note that most of these summaries follow this style; there are a few which are more like what you've written, but those should probably be fixed to fit this style.

Anyway, what I'm saying is, let's leave the original summary here. :)